lms/templates/courseware/dates.html

```diff
@@ -30,19 +30,46 @@ <h2 class="hd hd-2 date-title">
                 % if has_locked_assignments and verified_upgrade_link:
                     <div class="upgrade-banner">
                         <div class="upgrade-banner-text">
-                            <strong>${_('You are auditing this course.')}</strong>
-                            ${_('This means that you are unable to participate in graded assignments.')}
-                            ${_('To complete graded assignments as part of this course, you can upgrade today.')}
+                            <strong>${_('You are auditing this course,')}</strong>
+                            % if display_reset_dates_text:
+                                % if course_mode == CourseMode.AUDIT:
+                                    ${_('It looks like you missed some important deadlines based on our suggested schedule. To complete graded assignments as part of this course and shift the past due assignments into the future, you can upgrade today.')}
+                                % else:
+                                    ${_('It looks like you missed some important deadlines based on our suggested schedule. To keep yourself on track, you can update this schedule and shift the past due assignments into the future. Don’t worry—you won’t lose any of the progress you’ve made when you shift your due dates. ')}
```

Comment:
I don't agree with this having an em dashe in this but sure

Reply:
I reached out to Hilary & Eugene yesterday about this (whether it should be single hyphen, double hyphen, em dash, etc.). They confirmed to use the em dash. I wasn't sure if there were any technical reasons not to use it.